### DISCUSSION TITLE
mac m1 pro 导出pag卡住没反应

### DISCUSSION THREAD

**Discussion by xiazefeng** [Tencent/libpag#2127] [Q&A]
本人新手入门，刚装的2024的AE，无法打字，然后换成2023的AE正常使用，也安装了一些其它插件。通过PAGViewer 安装的AE插件。
然后在AE里随便建了一个形状图层，导出PAG files 就会卡住转圈圈，多次尝试都是这样，只能强制关闭AE。
想知道这是怎么回事。

**Comment by jinwuwu001** [comment]
请问该问题是否还存在，若存在请描述下复现路径和场景，包括系统版本、AE版本、PAGViewer版本等。

**Comment by jinwuwu001** [comment]
请问该问题是否还需要跟进，若需要请提供上述信息

**Comment by jinwuwu001** [comment]
用户长期无进一步反馈，暂时关闭，有需要可以再开启
